web/.eslintignore

```diff
@@ -0,0 +1 @@
+public

```

Comment:
Is it possible to add exclude by arguments, instead of a file? It's just a one line, so I think it's better to keep it in package.json :-)